superset/config.py

```diff
@@ -671,9 +671,8 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
 # Set this API key to enable Mapbox visualizations
 MAPBOX_API_KEY = os.environ.get("MAPBOX_API_KEY", "")
 
-# Maximum number of rows returned from a database
-# in async mode, no more than SQL_MAX_ROW will be returned and stored
-# in the results backend. This also becomes the limit when exporting CSVs
+# Maximum number of rows returned from a database, both SQL Lab, chart data
+# requests and CSV exports
```

Comment:
is it safe to say, on any query for a registered db connection?

Reply:
Thx - reworded as "Maximum number of rows returned for any analytical database query"